src/main/kotlin/blackjack/domain/Card.kt

```diff
@@ -1,14 +1,14 @@
 package blackjack.domain
```

Comment:
관련된 클래스들을 패키지로 모아서 관리할 수 있도록 해보세요.

```suggestion
package blackjack.domain.card
```